site/content/en/_index.html

```diff
@@ -55,7 +55,7 @@
 We do a [Pull Request](https://github.com/GoogleCloudPlatform/open-match/pulls) contributions workflow on **GitHub**. New users are always welcome!
 {{% /blocks/feature %}}
 
-{{% blocks/feature icon="fab fa-twitter" title="Follow us on Twitter!" url="https://twitter.com/open-match" %}}
+{{% blocks/feature icon="fab fa-twitter" title="Follow us on Twitter!" url="https://twitter.com/" %}}
```

Comment:
What's the point of a follow us on twitter link which just links to twitter.com?

Reply:
We are in the process of getting a twitter handle the test fails if I didn't make the change.